kickstarts/partials/post/bundler.ks.erb

```diff
@@ -18,3 +18,28 @@ popd
 # Install mime-types outside of bundle, needed by 'rest-client' gem when run under irb/automate
 # Run this command after 'bundle clean' so it won't be removed by 'bundle clean'
 gem install mime-types -v 2.6.1
+
+## Gem clean up
+# Unnecessary gem cache, will be rebuilt as needed
+rm -rf $(gem env gemdir)/cache/*
```

Comment:
Since we want to wipe out the contents of entire directory, should we be using tmpfs?

Reply:
Done